src/include/common/types/ku_string.h

```diff
@@ -65,7 +67,7 @@ struct ku_string_t {
     }
 
     std::string getAsShortString() const;
-    std::string getAsString() const;
+    KUZU_API std::string getAsString() const;
```

Comment:
TBH I might just export the entire class and same for value vector. We can check DuckDB to see their behaviour.